src/models/workOrder.test.js

```diff
@@ -96,7 +96,20 @@ describe('WorkOrder', () => {
 
       MockDate.set(subMinutes(now, 1))
 
-      expect(workOrder.appointmentStartTimePassed()).toBe(false)
+      expect(workOrder.appointmentISODatePassed()).toBe(true)
+    })
+
+    it('returns true when the current time is before the appointment start time', () => {
```

Comment:
Should this be `returns false when the current date is before the appointment date` ?